recipes/rnaprot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "RNAProt" %}
+{% set version = "0.3" %}
+{% set sha256 = "18ffa9d569c63e3fc5d7a31427c7569d6021c67587a6838aea4ff3af7f0780e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BackofenLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pytorch-cpu 1.8  # [cuda_compiler_version == "None"]
+    - pytorch-gpu 1.8  # [cuda_compiler_version != "None"]
+    - seaborn
+    - viennarna
+    - bedtools
+    - logomaker
+    - hpbandster
+    - markdown 3.2.2
```

Comment:
Hm, what is so special here?

Reply:
You mean markdown? With the newer version my markdown does not render correctly to html anymore. Don't no why so far ...